src/apps/concepts/components/MappingsTableRow.tsx

```diff
@@ -79,6 +79,15 @@ const conceptCodeFromUrl = (url: string): string => {
   return letters.join("");
 };
 
+const conceptFromUrl = (url: string): string => {
+  let letters = url.split("/");
+  letters.reverse();
+  letters.splice(0, 3, '');
+  letters.reverse();
+
+  return letters.join("/");
+};
```

Comment:
This could be done more efficiently using `slice()` (it also results in no array copies)

```suggestion
const conceptFromUrl = (url: string) => url.split("/").slice(0, -3).join("/") + "/";
```